meta-intel-edison-bsp/recipes-kernel/bcm43340/bcm43340-fw.bb

```diff
@@ -10,7 +10,7 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/files/:"
 
 SRC_URI = "git://github.com/01org/edison-firmware.git;branch=master;protocol=https;rev=8585a10b3527666b2d35b3dcacffede3ec00cb53;destsuffix=edison-firmware \
            git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git;branch=main;protocol=https;rev=1ffb7381a483a95d4bff923e7a73204dea0f3a66;destsuffix=linux-firmware \
-           git://git.osdn.net/gitroot/android-x86/device-generic-firmware.git;branch=nougat-x86;protocol=https;rev=afd71f20e36112edd8b1ad88f8055051069fd921;destsuffix=android-firmware \
+           git://github.com/htot/device-generic-firmware.git;branch=nougat-x86;protocol=https;rev=afd71f20e36112edd8b1ad88f8055051069fd921;destsuffix=android-firmware \
```

Comment:
But shouldn't this to point to google one to begin with?

Reply:
This (osdn) was the only place I could find a newer version than the one we already had. But seems the osdn server is being disrupted a lot lately, so I just cloned it.

Reply:
Newer than https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/brcm?id=5ac5ad5c882f88ed881174861934f184ff0173ff?

Reply:
More interesting is this one: https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/brcm?id=0f0aefd733f70beae4c0246edbd2c158d5ce974c